diff --git a/src/Toolbar.php b/src/Toolbar.php
index f4c717c..7a4d924 100644
--- a/src/Toolbar.php
+++ b/src/Toolbar.php
@@ -127,7 +127,7 @@ class Toolbar {
 
     $items['workspace_switcher']['tray'] = [
       '#heading' => $this->t('Switch to workspace'),
-      '#pre_render' => ['workspace.toolbar:preRenderWorkspaceSwitcherForms'],
+      //'#pre_render' => ['workspace.toolbar:preRenderWorkspaceSwitcherForms'],
       // This wil get filled in via pre-render.
       'workspace_forms' => [],
       'create_link' => $create_link,
